javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,16 @@
+var fullName = 'Nhu Pham';
+var isCool = true;
+var totalPets = 3;
+console.log('value of fullName:', fullName);
+console.log('value of isCool', isCool);
+console.log('value of totalPets:', totalPets);
+console.log('type of', typeof 'Nhu Pham');
+console.log('type of', typeof true);
+console.log('type of', typeof 3);
```

Comment:
You should be using the variables you created, not writing the values in yourself.